tests/dummy/app/templates/application.hbs

```diff
@@ -608,6 +607,170 @@
+              </main>
+            </div>
+          </div>
+        {{/freestyle-usage}}
+        {{#freestyle-usage "permanent-drawer-below"}}
+          <div class="component-doc flexible">
+            <header class="mdc-toolbar mdc-toolbar--fixed fixed-relative">
+              <div class="mdc-toolbar__row">
+                <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
+                  <span class="catalog-back">
+                    <a href="#" class="mdc-toolbar__icon--menu"><i class="material-icons"></i></a>
+                  </span>
+                  <span class="mdc-toolbar__title catalog-title">Permanent Drawer Below Toolbar</span>
+                </section>
+              </div>
+            </header>
+            {{#mdc-drawer as |drawer|}}
+              {{#drawer.toolbar-spacer}}{{/drawer.toolbar-spacer}}
+              {{#drawer.content}}
+                {{#mdc-list-group as |group|}}
+                  {{#mdc-list tagName='nav' dense=true as |list|}}
+                    {{#list.item class="mdc-permanent-drawer--selected"}}<i class="material-icons mdc-list-item__start-detail" aria-hidden="true">inbox</i>Inbox{{/list.item}}
+                    {{#list.item}}<i class="material-icons mdc-list-item__start-detail" aria-hidden="true">star</i>Star{{/list.item}}
+                    {{#list.item}}<i class="material-icons mdc-list-item__start-detail" aria-hidden="true">send</i>Sent Mail{{/list.item}}
+                    {{#list.item}}<i class="material-icons mdc-list-item__start-detail" aria-hidden="true">drafts</i>Drafts{{/list.item}}
+                  {{/mdc-list}}
+                  {{group.divider}}
+                  {{#mdc-list tagName='nav' dense=true as |list|}}
+                    {{#list.item}}<i class="material-icons mdc-list-item__start-detail" aria-hidden="true">email</i>All Mail{{/list.item}}
+                    {{#list.item}}<i class="material-icons mdc-list-item__start-detail" aria-hidden="true">delete</i>Trash{{/list.item}}
+                    {{#list.item}}<i class="material-icons mdc-list-item__start-detail" aria-hidden="true">report</i>Spam{{/list.item}}
+                  {{/mdc-list}}
+                {{/mdc-list-group}}
+              {{/drawer.content}}
+            {{/mdc-drawer}}
+            <div class="demo-content inner-flexible">
+              <main class="demo-main">
+                <h1>Permanent Drawer</h1>
+                <p>It sits to the left of this content.</p>
+              </main>
+            </div>
+          </div>
+        {{/freestyle-usage}}
+      {{/section.subsection}}
+      {{#section.subsection name="Persistent Drawers"}}
+        {{#freestyle-usage "persistent-drawer"}}
+          <div class="component-doc flexible">
+            {{#mdc-drawer tagName='aside' persistent=true permanent=false open=isPersistentDrawerOpened as |drawer|}}
+              {{#drawer.toolbar-spacer}}{{/drawer.toolbar-spacer}}
```

Comment:
The nested components should know whether they are permanent, persistent, or permanent as well to get appropriate classes.